src/vm/codeversion.cpp

```diff
@@ -518,9 +518,11 @@ ILCodeVersionNode::ILCodeVersionNode() :
     m_rejitId(0),
     m_pNextILVersionNode(dac_cast<PTR_ILCodeVersionNode>(nullptr)),
     m_rejitState(ILCodeVersion::kStateRequested),
-    m_pIL(dac_cast<PTR_COR_ILMETHOD>(nullptr)),
+    m_pIL(),
```

Comment:
I moved this because the constructor of VolatilePtr ends up trying to assign ((volatile PTR_COR_IL_METHOD &)m_val) = val; and complains because PTR_COR_IL_METHOD doesn't have a conversion to volatile. The store method just calls VolatileStore on it, so is fine.